packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { composeStories } from '@storybook/react';
+import * as accordionStories from './Accordion.stories';
+import { Box } from '~components/Box';
+import { Heading } from '~components/Typography';
+
+const allStories = Object.values(composeStories(accordionStories));
+
+export const Accordion = (): JSX.Element => {
```

Comment:
in react-native do these kitchen sinks exists? If so i hope they won't pollute the sidebar unnecessarily 😢

Reply:
but that shouldn't matter na because Kitchen Sink is a directory and it will come at the last

Reply:
Oh accha 6.5 mein we have nice directories in RN also, okay.